### DISCUSSION TITLE
W5500 and mcp23s08 possible?

### DISCUSSION THREAD

**Discussion by stsch83** [arendst/Tasmota#21501] [Q&A]
Hello,
is there a way to use both, W5500 and mcp23s08 in the same project. Both features needs SPI to communicate with the esp32.
In my example mcp23s08 is running fine. But i'm not able to get w5500 working.

BR,
Stefan

**Comment by sfromis** [comment]
Ideally, devices can share the same SPI bus, with each having their own CS pin. I'd suggest first getting W5500 to work without the MCP23S08, and only try to combine after that. Note that I'm not 100% sure if the Ethernet driver can share the SPI bus for the W5500. Tasmota does also have some support for two SPI busses, but neither driver seems to be among the few supporting this enhancement.
